website/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1">
+    <title>Document</title>
+    <link rel="stylesheet" href="./out/index.css">
+</head>
+<body>
+    
+    <header>
+        <img class="logo to-main-page" src="assets/logo_white.png" />
+        <a href="#" class="download">Download</a>
+        <a href="#" >We are open source!</a>
+        <a href="#" >Support us!</a>
```

Comment:
you forgot to add the hrefs (idk if this was intended since it is uncomplete)

Reply:
these are placeholder links